src/main/java/com/teamwizardry/librarianlib/core/common/OreDictionaryRegistrar.kt

```diff
@@ -17,6 +18,9 @@ import net.minecraftforge.oredict.OreDictionary
 @Mod.EventBusSubscriber(modid = LibrarianLib.MODID)
 object OreDictionaryRegistrar {
     val toRegister = mutableMapOf<() -> ItemStack, String>()
+    val furnaceRecipeStacks = mutableMapOf<() -> ItemStack, Pair<() -> ItemStack, Float>>()
+    val furnaceRecipeItems = mutableMapOf<() -> Item, Pair<() -> ItemStack, Float>>()
+    val furnaceRecipeBlocks = mutableMapOf<() -> Block, Pair<() -> ItemStack, Float>>()
```

Comment:
Any reason these 4 (including `toRegister`) are maps and not `List`s of `Pair`s ?
I think they should be `LinkedList`s ideally.
Also, shouldn't all 4 be `private` too?